src/main/java/seedu/address/model/person/ReadOnlyPerson.java

```diff
@@ -49,17 +51,19 @@ default boolean isSameStateAs(ReadOnlyPerson other) {
     default String getAsText() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName())
-                .append(" Phone: ")
+                .append(" | Phone: ")
```

Comment:
Why the | symbol here?

Reply:
After every add/edit persons, I realized the person details shown is too cluttered. It's one whole long string of text without any separator in between each field resulting in things like this:

... Favorite: No Display Photo: null ...

It kind of reads like "No Display Photo" but that's not the case. What do ya think?

Reply:
Sounds good!